facturacion_mexico/facturacion_fiscal/timbrado_api.py

```
@@ -3241,106 +3376,277 @@ def _cascade_cancel_previous_after_substitute(new_ffm_name: str):
 		return {"cascade": "error", "message": str(e)}
 
 
+def _mark_substitution_cancellation_pending(orig_ffm_name, orig_si_name, error=None):
+	"""Deja A en PENDIENTE_CANCELACION (fiscal) + fm_sync_status='pending' + fm_sync_error, SIN tocar
+	docstatus ni el timbrado de B. Reutiliza apply_cancellation_state (writer autoritativo). El
+	scheduler `retry_pending_substitution_cancellations` la retomará (durabilidad vía estado en BD)."""
+	from facturacion_mexico.facturacion_fiscal.cancellation_state import apply_cancellation_state
+
+	try:
+		apply_cancellation_state(orig_ffm_name, FiscalStates.PENDIENTE_CANCELACION, sync_status="pending")
+	except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)